src/renderer/components/+add-cluster/add-cluster.tsx

```diff
@@ -189,20 +188,6 @@ export class AddCluster extends React.Component {
           When connecting to a cluster, make sure you have a valid and working kubeconfig for the cluster. Following lists known "gotchas" in some authentication types used in kubeconfig with Lens
           app.
         </p>
-        <a href="https://kubernetes.io/docs/reference/access-authn-authz/authentication/#option-1-oidc-authenticator" target="_blank">
-          <h3>OIDC (OpenID Connect)</h3>
-        </a>
```

Comment:
are we no longer supporting OIDC?

Reply:
Yes, but the gotcha is outdated now, since kubeconfigs are file references